homework-submission/homework-week1/app.js

```diff
@@ -0,0 +1,160 @@
+const Item = ({ id, description, deadline, done, removeItem }) => {
+  return (
+    <li key={id} className={done === true ? 'true' : ''}>
```

Comment:
No need for a key here, while you have a one in it's correct spot at the very first level of the array `.map` generates!
```suggestion
    <li className={done === true ? 'true' : ''}>
```

You have className-s such as `true` or `ok`, they are fine, But I would go for another class names meaningful css-like instead